<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepJudge</class>
 <widget class="QDialog" name="StepJudge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/Top.qrc">
    <normaloff>:/new/wl/Logo_ico.png</normaloff>:/new/wl/Logo_ico.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>731</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>721</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBoxDate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>715</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnRPNSort">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>422</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RPN风险排序</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>423</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnFilterTOPRisk">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>423</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤TOP(N)风险项</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnShowAll">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>423</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示所有风险项</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>456</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请指定FEMA措施评审专家：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditFEAMExpert">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>721</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/Top.qrc"/>
 </resources>
 <connections/>
</ui>
